### PR TITLE
influxdb 1.0.0-beta1

### PR DESCRIPTION
**Pull request by gunnaraasen**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

This is the beta release in preparation for hitting 1.0. It is more akin to a full minor release than true beta, hence making this the default version instead of a `--devel` release.

### CODE REVIEW

**Comment by ilovezfs** [comment]
:-1: See comments here: https://github.com/Homebrew/homebrew-core/pull/1802#issuecomment-224524007

**Comment by DomT4** [comment]
I'm fine with us taking this, we've plenty of form for doing so, but it should be inside a `devel do` block as suggested by ILZ.

**Comment by gunnaraasen** [comment]
@ilovezfs @DomT4 thanks for the feedback and sorry for the delay in getting this updated. I've switched the beta to `devel`.

File: Formula/influxdb.rb [hunk 1]
```diff
@@ -42,6 +47,11 @@ class Influxdb < Formula
     :revision => "fc32781af5e85e548d3f1abaf0fa3dbe8a72495c"
   end
 
+  go_resource "github.com/dgrijalva/jwt-go" do
+    url "https://github.com/dgrijalva/jwt-go.git",
+    :revision => "a2c85815a77d0f951e33ba4db5ae93629a1530af"
```
Comment (ilovezfs) [inline Formula/influxdb.rb]: Should this be bumped to f0777076321ab64f6efc15a82d9d23b98539b943?

**Comment by gunnaraasen** [comment]
Opened up a new PR to target the most recent version, see https://github.com/Homebrew/homebrew-core/pull/2212.

**Pull request closed without merging**